submissions/opeyemiPeter/Javascript/task-2.js

```diff
@@ -0,0 +1,19 @@
+const diff = 123 - 7;
+var  num1 = 1;
+
+while (num1 < diff) {
```

Comment:
This statement should be less than and equal to @opeyemipeter1759

Reply:
i will correct them now thanks @ToluAkin